tests/testthat/test_aggregate_sex.R

```diff
@@ -0,0 +1,51 @@
+testthat::context("aggregate_sex() tests")
+
+# set up standard input data.table
+input_dt <- data.table::CJ(location = "France", year = 2010:2020,
+                           sex = c("female", "male"),
+                           value1 = 1, value2 = 2)
+id_cols <- c("location", "year", "sex")
+value_cols <- c("value1", "value2")
+setkeyv(input_dt, id_cols)
+
+# set up expected output table
+expected_dt <- data.table::CJ(location = "France", year = 2010:2020,
+                              sex = c("both"),
+                              value1 = 2, value2 = 4)
+expected_dt <- rbind(input_dt, expected_dt, use.names = T)
+setkeyv(expected_dt, id_cols)
+
+test_that("check `aggregate_sex()` basic functionality works", {
```

Comment:
I think test names should follow a more readable pattern, so that you could say "When this test is run, it will test that...". Also, do we need to say the name of the function when the test is already associated with that file?

Reply:
Some of the test files contain tests for more than one function, mirroring the files in the `R/` directory. In those cases especially I think its helpful to include the function name.

Here is an example error message when one of the tests fail. I think short and clear test contexts make sense. I don't think "When this test is run" adds extra information.

Thoughts? @spencerpease @krpaulson27 
```
x |  17 1     | test_calculate_age [0.2 s]
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
test_calculate_age.R:100: failure: check `calculate_age_name()` basic functionality works
`calculate_age_name(input_dt, format = "interval")` did not throw an error.
```

Reply:
I didn't mean to actually to include the phrase "When this test is is run", just that I think it is better to use a present/imperative tense when naming test, so that when running tests the output looks more like a to-do list.

Maybe a better way to think of it is `test_that()` should be the first two words of the phrase that the test name completes.

I do think it is useful to include the name of the function when testing a file with many functions in it though, so in this example it would be `test_that("aggregate_sex() basic functionality works", ...`